pkgs/native_toolchain_c/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.18.0-wip
+## 0.18.0
```

Comment:
Duplicate changelog entry

Also, this could be 0.17.1 right? We didn't change anything in the API, just bumped dependencies.